frontend/src/metabase/palette/hooks/useCommandPaletteBasicActions.tsx

```diff
@@ -186,7 +187,18 @@ export const useCommandPaletteBasicActions = ({
       },
     ];
 
-    return [...actions, ...browseActions];
+    const diagnosticAction = {
+      id: "diagnostic_modal",
+      name: t`Open error diagnostic modal`,
+      section: "basic",
+      icon: "info",
+      shortcut: ["$mod+f1"],
```

Comment:
this shortcut definition is so much cleaner!

Reply:
🥳